.github/workflows/docker-openmetadata-server.yml

```diff
@@ -40,5 +44,5 @@ jobs:
           platforms: linux/amd64,linux/arm64
           push: ${{ github.event_name == 'release' }}
           # Update tags before every release
-          tags: 'openmetadata/server:0.10.0,openmetadata/server:latest'
+          tags: 'openmetadata/server:${{ github.event.inputs.tag }},openmetadata/server:latest'
```

Comment:
`github.event.inputs.tag` will be null in case of Github CI triggered by Release published event!